src/common/dataAdaptors/songAdaptors.js

```diff
@@ -32,6 +32,8 @@ export function songForViewing({ item }) {
     ...introAdaptor({ item }),
     ...notesAcknowledgementsAdaptor({ item }),
     ...relatedMediaForViewing({ item }),
+    createdBy: item?.createdBy,
```

Comment:
If you move these into the `basicDatesAdaptor` they will be reusable for FW-6311 stories.